Library/Formula/x265.rb

```diff
@@ -13,20 +14,17 @@ class X265 < Formula
     sha256 "440805e02452b80400ff428967bd4c5d24f042bea2cc6b5d76a3a37648965a60" => :mountain_lion
   end
 
-  head 'https://bitbucket.org/multicoreware/x265', :using => :hg
+  option "with-16-bit", "Build a 16-bit x265 (default: 8-bit)"
 
-  depends_on 'yasm' => :build
-  depends_on 'cmake' => :build
-  depends_on :macos => :lion
+  deprecated_option "16-bit" => "with-16-bit"
```

Comment:
Can one/any of @Homebrew/owners chime in on this? Are we content keeping the `16-bit`, `32-bit` options or do we want to migrate them to `with/without`?